nise/report.py

```diff
@@ -731,9 +732,11 @@ def aws_create_report(options):  # noqa: C901
 
 def azure_create_report(options):  # noqa: C901
     """Create a cost usage report file."""
-    data = []
     start_date = options.get("start_date")
     end_date = options.get("end_date")
+    row_limit = options.get("row_limit")
+    if not row_limit:
+        row_limit = 10_000_000  # to avoid unintended splitting when flag is not passed
```

Comment:
I'm not sure why we need this? The default row_limit is 100,000. Are we generating reports larger than that? But either way, let's just not worry about this. If someone does not want split reports, then they can provide a higher limit thru the cli.